src/Http/Routing/src/Matching/JumpTableBuilder.cs

```diff
@@ -85,13 +86,15 @@ public static JumpTable Build(int defaultDestination, int exitDestination, (stri
         }
 
         // Use the ILEmitTrieJumpTable if the IL is going to be compiled (not interpreted)
-        if (RuntimeFeature.IsDynamicCodeCompiled)
+        return MakeILEmitTrieJumpTableIfSupported(defaultDestination, exitDestination, pathEntries, fallback);
```

Comment:
I think this change is unnecessary and can be reverted. The existing code should be able to pass CI.

Reply:
I'll leave it as is. It will be changed again when analysis supports IsDynamicCodeCompiled.